app/controllers/ChoiceController.scala

```diff
@@ -59,7 +59,6 @@ class ChoiceController @Inject()(
             validChoice.value match {
               case SupplementaryDec | StandardDec =>
                 Redirect(controllers.declaration.routes.DispatchLocationPageController.displayPage())
-              case Arrival | Departure => Redirect(controllers.movement.routes.MovementController.displayDucrPage())
```

Comment:
Hmm, so now when you choose arrival or departure the page will refresh. Do we have ticket to redirect to another microservice?

Because view still contains those radio buttons

Reply:
Ah need to change the view... to remove those options... will do this